app/views/alaveteli_pro/account_request/new.html.erb

```diff
@@ -3,8 +3,13 @@
     <h1>WhatDoTheyKnow<sup>Pro</sup> is a powerful, fully-featured <strong class="marketing-highlight">FOI toolkit</strong> for journalists</h1>
   <h2>Everything you need to keep on top of complex FOI-driven stories.</h2>
   <div class="marketing__hero__cta">
-    <h3>Now in beta </h2>
-    <a href="#input-beta" class="button" id="launch-access">Get pre-launch access</a>
+    <% if feature_enabled?(:pro_pricing) %>
+     <h3>Now in public beta</h3>
+     <%= link_to _('Sign up'), pro_plans_path, :class => "button" %>
```

Comment:
> TODO - find right wording for call to action

"Plans and pricing" is the title of the next page.

@wrightmartin any idea here?

![screen shot 2017-10-06 at 17 01 52](https://user-images.githubusercontent.com/282788/31287113-1ec00b9c-aab8-11e7-9750-44e4c259dbe6.png)